### ISSUE TITLE
is it possible to check cookie exist?

### ISSUE THREAD

**Issue by mrmcan** [kyprizel/testcookie-nginx-module#57]
Hi, 

I would like to check if cookie exist, dont set again. Is it possible to do it?

**Comment by Mecanik** [comment]
No need to do that, just leave default expiry date. It will expire in years...

**Comment by khaleghsalehi** [comment]
Yes, I fixed it here  (ironfox is a fork of testcookie-nginx-module)
https://github.com/khaleghsalehi/ironfox/blob/master/modules/ngx_http_bot_protection_module/src/ngx_http_bot_protection_module.c#L1632
